db/migrate/20170524011352_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, foreign_key: true
+      t.boolean :like
```

Comment:
makes sense - i don't think i'll aim for keeping track of the likes at this stage!